src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -118,17 +124,18 @@ public RobotMap() {
 		Component.rightWheel.disablePID(); // TODO add encoders
 		Component.chassis = new TankDrive("StrongholdChassis", Component.leftWheel, Component.rightWheel);
 		// Intake
-		Component.intakeRoller = new VelocityEncodedMotor("topIntakeRoller", new AccelerationCap(Component.pdp), new CANEncoder(Port.Sensors.intakeEncoder), new CANTalon(CAN.intakeRoller));
+		Component.intakeRoller = new Innie(new CANTalon(CAN.intakeRoller), new CANEncoder(Port.Sensors.intakeEncoder));
```

Comment:
Please move the CANTalon argument to the end to be consistent with VelocityEncodedMotor